### ISSUE TITLE
DemoMEV exceeds maximum update depth when Model is changed

### ISSUE THREAD

**Issue by rod-glover** [pacificclimate/pcic-react-components#3]
This is new behaviour. What has changed is that I did a fresh reinstall. Now this demo implodes.

**Comment by rod-glover** [comment]
According to package-lock.json, very little has changed. To wit:

```
Index: package-lock.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- package-lock.json	(revision ff25c66745e5d891ac2c6b2e072fc7f656b8381b)
+++ package-lock.json	(date 1553558307000)
@@ -1,6 +1,6 @@
 {
   "name": "pcic-react-components",
-  "version": "1.1.0",
+  "version": "1.1.1",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
@@ -3599,7 +3599,8 @@
       "version": "1.1.0",
       "resolved": "https://registry.npmjs.org/console-control-strings/-/console-control-strings-1.1.0.tgz",
       "integrity": "sha1-PXz0Rk22RG6mRL9LOVB/mFEAjo4=",
-      "dev": true
+      "dev": true,
+      "optional": true
     },
     "constants-browserify": {
       "version": "1.0.0",
@@ -16137,12 +16138,14 @@
             "balanced-match": {
               "version": "1.0.0",
               "bundled": true,
-              "dev": true
+              "dev": true,
+              "optional": true
             },
             "brace-expansion": {
               "version": "1.1.11",
               "bundled": true,
               "dev": true,
+              "optional": true,
               "requires": {
                 "balanced-match": "^1.0.0",
                 "concat-map": "0.0.1"
@@ -16162,7 +16165,8 @@
             "concat-map": {
               "version": "0.0.1",
               "bundled": true,
-              "dev": true
+              "dev": true,
+              "optional": true
             },
             "console-control-strings": {
               "version": "1.1.0",
@@ -16284,7 +16288,8 @@
             "inherits": {
               "version": "2.0.3",
               "bundled": true,
-              "dev": true
+              "dev": true,
+              "optional": true
             },
             "ini": {
               "version": "1.3.5",
@@ -16310,6 +16315,7 @@
               "version": "3.0.4",
               "bundled": true,
               "dev": true,
+              "optional": true,
               "requires": {
                 "brace-expansion": "^1.1.7"
               }
```

**Comment by rod-glover** [comment]
Breaks when you select

FGOALS-g2
GFDL-CM3
GFDL-ESM2M
HadGEM2-A0
MIROC-ESM-CHEM
bcc-csm1-1-m

but not

ACCESS1-0 (default)
CCSM4
CNRM-CM5
CSIRO-Mk3-6-0
CanESM2
GFDL-ESM2G
HadGEM2-CC
HadGEM2-ES
MIROC5
MPI-ESM-LR
MRI-CGCM3
inmcm4

**Comment by rod-glover** [comment]
Also get maximum update depth exceeded when selecting any other Emissions Scenario when CCSM4 (which allows all 3).

Breakage does not occur when selecting anything using DemoSimple. So breakage is likely due to cascading of selector constraints.

**Comment by rod-glover** [comment]
Fixed in #6
